Tests/GRPCHTTP2CoreTests/Client/GRPCClientStreamHandlerTests.swift

```diff
@@ -19,10 +19,10 @@ import NIOCore
 import NIOEmbedded
 import NIOHPACK
 import NIOHTTP1
-import NIOHTTP2
 import XCTest
 
 @testable import GRPCHTTP2Core
+@testable import NIOHTTP2
```

Comment:
We really shouldn't do this because we're taking a dependency on the internal API of a module we don't own.

Reply:
Hm yes true - we should probably make the constructor public on NIO though, otherwise this API is unusable.